src/test/fuzz/utxo_snapshot.cpp

```diff
@@ -104,7 +104,7 @@ void utxo_snapshot_fuzz(FuzzBufferType buffer)
                 outfile << coinbase->GetHash();
                 WriteCompactSize(outfile, 1); // number of coins for the hash
                 WriteCompactSize(outfile, 0); // index of coin
-                outfile << Coin(coinbase->vout[0], height, /*fCoinBaseIn=*/1);
+                outfile << Coin(coinbase->vout[0], height, /*fCoinBaseIn=*/true);
```

Comment:
Can a current utxo snapshot still be loaded with this change?

Reply:
I looked at the changes more closely now, and yes, this should not change the format. I also checked that doing a `dumptxoutset` produces the same data on both master and this PR.